Network/Converter/IPacketConverter.cs

```diff
@@ -1,56 +1,67 @@
-﻿#region Licence - LGPLv3
-// ***********************************************************************
-// Assembly         : Network
-// Author           : Thomas
-// Created          : 07-24-2015
-//
-// Last Modified By : Thomas
-// Last Modified On : 03-10-2016
-// ***********************************************************************
-// <copyright>
-// Company: Indie-Dev
-// Thomas Christof (c) 2016
-// </copyright>
-// <License>
-// GNU LESSER GENERAL PUBLIC LICENSE
-// </License>
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as published by
-//  the Free Software Foundation, either version 3 of the License, or
-//  (at your option) any later version.
-//
-//  This program is distributed in the hope that it will be useful,
-//  but WITHOUT ANY WARRANTY; without even the implied warranty of
-//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//  GNU General Public License for more details.
-//
-//  You should have received a copy of the GNU Lesser General Public License
-//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-// ***********************************************************************
-#endregion Licence - LGPLv3
-
+﻿using Network.Packets;
 using System;
 
 namespace Network.Converter
 {
     /// <summary>
-    /// Provides extension methods for packets to handle their read and write behaviors.
+    /// Describes the methods that a packet converter must implement in order
+    /// to be able to serialise and deserialise packets to and from a binary
+    /// form.
     /// </summary>
     public interface IPacketConverter
     {
+        #region Methods
+
         /// <summary>
-        /// Converts a given packet to a byte array.
+        /// Serialises a given packet to a byte array.
         /// </summary>
         /// <param name="packet">The packet to convert.</param>
         /// <returns>System.Byte[].</returns>
-        byte[] GetBytes(Packet packet);
+        byte[] SerialisePacket(Packet packet);
```

Comment:
Again. Public class and u did change the interface.
You can rename it, but u have to support the old one as well.